docs/release-notes/identify-task.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Bug Fixes**
+
+-  API: Fix a bug where a database query would add extra quotes around the task type.
```

Comment:
I would remove this first line about database query part, and just keep it to behavior that users will see

sicne if I'm a user i don't care about the cause of the bug just how it affects me